src/column/ColumnManager.ts

```diff
@@ -149,6 +152,24 @@ export default class ColumnManager extends EventHandler {
     this.$parent.selectAll('.column').remove();
   }
 
+  private setWidthToURLHash() {
+    hash.setProp('colWidths',
+      this.filtersHierarchy
+        .map((d) => d.width.toFixed())
+        .join(ColumnManager.HASH_FILTER_DELIMITER)
+    );
+  }
+
+  private initWidthFromURLHash(column:AColumn<any,any>) {
+    if (hash.has('colWidths')) {
+      const widths = hash.getProp('colWidths')
+        .split(ColumnManager.HASH_FILTER_DELIMITER);
+
+      const width = parseFloat(widths[this.columns.indexOf(column)]);
```

Comment:
shouldn't it be  based on the `filterHierarchy`? since this will also be used for persisting the width?